metadata-ingestion/src/datahub/ingestion/source/sql/sql_common.py

```diff
@@ -484,6 +494,16 @@ def get_workunits_internal(self) -> Iterable[Union[MetadataWorkUnit, SqlWorkUnit
             profile_requests: List["GEProfilerRequest"] = []
             if sql_config.profiling.enabled:
                 profiler = self.get_profiler_instance(inspector)
+                try:
+                    self.add_profile_metadata(inspector)
+                except Exception as e:
+                    logger.error(
+                        f"Failed to get enrichment data for profiler \n{traceback.format_exc()}",
+                    )
+                    self.report.report_failure(
+                        "profile_metadata",
+                        f"Failed to get enrichment data for profile {e}",
+                    )
```

Comment:
Same thing here -- do we want to prevent stale entity deletion because we couldn't get this data?